Resources/Prototypes/_RMC14/Catalog/Cargo/requisitions_catalog.yml

```diff
@@ -122,6 +122,8 @@
 #        crate: CMCrateInflatableBarriers
       - cost: 2000
         crate: CMCrateReplacementLights
+      - cost: 2500
+        crate: RMCCrateFloodlights
```

Comment:
There's no crate of floodlights that I can see in 13.
![image](https://github.com/user-attachments/assets/727c8660-0472-4bbf-8c57-ac9b41f0d1c9)

```suggestion
```